.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+install:
+- npm install
+- npm run bower:install
```

Comment:
the reason why it's working is because the nodeJS travis nodes has "./node_modules/.bin" in $PATH already